### ISSUE TITLE
Additional features?

### ISSUE THREAD

**Issue by KolbySisk** [KolbySisk/next-supabase-stripe-starter#4]
One of my goals when building this was to keep the project minimal. But I've had a few requests to add additional features. I'm curious if there is any consensus on adding additional features, and if so - which features would you like to see?

If you're reading this and you have an opinion on keeping this minimal vs adding features I'd love to hear your thoughts and which features you'd like to see included.

### Potential features:

- Testing (integration testing with Playwright and MSW)
- i18n (i18next?)
- Experimentation system (LaunchDarkly?)
- Error Monitoring (Sentry?)
- Observability (Better Stack?)
- Type-safe environment (T3 Env)

**Comment by uptoamir** [comment]
Hi @KolbySisk 
It is nice to have LaunchDarkly feature also.

**Comment by kluplau** [comment]
Yeah it's a hard thing to balance features and minimalism just right.

i18n and Observability would be huge tho, and I honestly don't think it will add too much noise.
LaunchDarkly would be nice, but I fear it will add too much noise. I haven't used any feature flag implementation, so don't depend solely on my take.

**Comment by jonleinena** [comment]
Would definitely love i18n! Been a bit of a pain to set up that feature

**Comment by traflagar** [comment]
When it comes to getting started with this, I'd like to alter the product cards bulletpoints. I'm pretty noob and could use some more clear instructions on how to add or remove or change the metadata entries (e.g. from generate images ## to a simple feature bulletpoint).

I tried removing the metadata as cleanly as I could and was still breaking the app.

The zod schema is hard for me to interpret or alter because it is set up in a pretty compact way that I just don't know how to get around.

Maybe a set of files that let the starter kit user simply fill in five features in bullet point.

Thanks for the app!

**Comment by dadof3bytes** [comment]
Please add teams/organizations so people can build multi-tenant apps for organizations with multiple users.
